java/system-test/system-test-drivers/src/main/java/sleeper/systemtest/drivers/metrics/AwsTableMetricsDriver.java

```diff
@@ -59,11 +59,11 @@
 public class AwsTableMetricsDriver implements TableMetricsDriver {
     private static final Logger LOGGER = LoggerFactory.getLogger(AwsTableMetricsDriver.class);
     private static final Map<String, String> METRIC_ID_TO_NAME = Map.of(
-            "activeFiles", "ActiveFileCount",
+            "activeFiles", "NumberOfFilesWithReferences",
             "records", "RecordCount",
             "partitions", "PartitionCount",
             "leafPartitions", "LeafPartitionCount",
-            "filesPerPartition", "AverageActiveFilesPerPartition");
+            "filesPerPartition", "averageFileReferencesPerPartition");
```

Comment:
This can be capitalised consistently with the other metrics.

Reply:
Comment addressed